src/ioq_sup.erl

```diff
@@ -47,3 +53,42 @@ get_ioq2_servers() ->
     lists:map(fun(I) ->
         list_to_atom("ioq_server_" ++ integer_to_list(I))
     end, lists:seq(1, erlang:system_info(schedulers))).
+
+handle_config_change("ioq", _Key, _Val, _Persist, St) ->
+    gen_server:cast(ioq_server, update_config),
+    {ok, St};
+handle_config_change("ioq2" ++ _, _Key, _Val, _Persist, St) ->
+    lists:foreach(fun({_Id, Pid}) ->
+        gen_server:call(Pid, update_config)
```

Comment:
I don't know. This is how `ioq` and `ioq2` implemented. 
- `ioq` [uses cast](https://github.com/apache/couchdb-ioq/blob/master/src/ioq_server.erl#L139)
- `ioq2` [uses call](https://github.com/apache/couchdb-ioq/blob/master/src/ioq_server2.erl#L308)

Changing the implementation of `ioq` and `ioq2` is out of scope of the PR.

Reply:
`ioq2` [also supports cast for config updates](https://github.com/apache/couchdb-ioq/blob/master/src/ioq_server2.erl#L324-L325) which is why I asked originally. However, since the tests pass, I don't see a strong motivation to change anything.

Reply:
Hmm. I didn't see it. In this case I tend to open a follow up PR to make it consistent with `ioq`.

Reply:
https://github.com/apache/couchdb-ioq/pull/9